BrainPortal/lib/portal_system_checks.rb

```diff
@@ -285,5 +285,26 @@ def self.z020_start_background_activity_workers #:nodoc:
 
   end
 
+
+
+  def self.z040_ensure_sti_enabled #:nodoc:
```

Comment:
replace "enabled" with "valid" because it's not like STI is something that can be enabled or disabled